### PR TITLE
ready for review: version control with github.md

### PR DESCRIPTION
**Pull request by sateeshperi**
adding lesson for version control with github

PR author:
- [x] pull request is ready for review & merge

Things to check:

- [x] tutorial resets working directory at beginning with a `cd ~/``
- [ ] tutorial contains relevant `conda install -y` commands at the beginning
- [ ] tutorial links to previous tutorial (at top) and next tutorial (at bottom)
- [x] tutorial has learning objectives at top

### CODE REVIEW

**Comment by sateeshperi** [comment]
add and update lesson for github

File: github.md [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+
+`git status`: Check the status of your repository. See which files are inside it, which changes still need to be committed, and which branch of the repository you’re currently working on.
+
+`git add`: This does not add new files to your repository. Instead, it brings new files to Git’s attention. After you add files, they’re included in Git’s “snapshots” of the repository.
+
+`git commit`: Git’s most important command. After you make any sort of change, you input this in order to take a “snapshot” of the repository. Usually it goes git commit -m “Message here.” The -m indicates that the following section of the command should be read as a message.
+
+`git branch`: Working with multiple collaborators and want to make changes on your own? This command will let you build a new branch, or timeline of commits, of changes and file additions that are completely your own. Your title goes after the command. If you wanted a new branch called “cats,” you’d type git branch cats.
```
Comment (taylorreiter) [inline github.md]: `git branch cats` in back ticks to format the code

File: github.md [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+
+`git checkout`: Literally allows you to “check out” a repository that you are not currently inside. This is a navigational command that lets you move to the repository you want to check. You can use this command as git checkout master to look at the master branch, or git checkout cats to look at another branch.
```
Comment (taylorreiter) [inline github.md]: `git checkout master` and `git checkout cats` in back ticks again

File: github.md [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+
+`git merge`: When you’re done working on a branch, you can merge your changes back to the master branch, which is visible to all collaborators. git merge cats would take all the changes you made to the “cats” branch and add them to the master.
```
Comment (taylorreiter) [inline github.md]: again, `git merge cats`

File: github.md [hunk 4]
```diff
@@ -0,0 +1,241 @@
...
+diblynn@js-17-71:~$
+```
+
+Make sure you're in your home directory:
+
+```
+cd ~
+```
+
+It’s time to introduce yourself to Git. Type in the following code:
```
Comment (taylorreiter) [inline github.md]: love this :)

**Review by taylorreiter — APPROVED** [review]
Really great, could be merged as is. I think a few code things could be surrounded in backticks, and you could increase the number of hashtags in the headers to improve the markdown rendering, but really clear! 

Is there any external resources that need to be cited?

**Comment by sateeshperi** [comment]
all sources have been cited in the resources section.

**Comment by taylorreiter** [comment]
Ok! Maybe make it clear that those were your sources as well?

**Pull request merged**